docs/pages/v2/governance.md

```diff
@@ -592,7 +592,7 @@ const tx = await gov.methods.castVoteBySig(proposalId, 1, v, r, s).send({});
 
 ## Timelock
 
-Each protocol contract is controlled by the Timelock contract, which can modify system parameters, logic, and contracts in a 'time-delayed, opt-out' upgrade pattern. The Timelock has a hard-coded minimum delay of 2 days, which is the least amount of notice possible for a governance action.
+Each protocol contract is controlled by the [Timelock contract](https://github.com/compound-finance/compound-protocol/blob/master/contracts/Timelock.sol){:target="_blank"}, which can modify system parameters, logic, and contracts in a 'time-delayed, opt-out' upgrade pattern. The Timelock has a hard-coded minimum delay of 2 days, which is the least amount of notice possible for a governance action.
```

Comment:
> The Timelock has a hard-coded minimum delay of 2 days, which is the least amount of notice possible for a governance action.

That is true for the mainnet governance timelock, but just as an FYI, the L2 Timelocks can have their minimum delay set to other values. For example, the minimum delay for the Polygon timelock is only 12 hours

Reply:
Good call, this isn't even true on mainnet anymore, I think it is one day there.